### PR TITLE
Add Wallet type support for meta actions

### PR DESCRIPTION
**Pull request by samuveth**

### CODE REVIEW

File: src/sign/index.ts [hunk 1]
```diff
@@ -106,11 +109,11 @@ export default class Client {
     return await this.sign(web3, address, message, type);
   }
 
-  async follow(web3: Web3Provider, address: string, message: Follow) {
+  async follow(web3: Wallet, address: string, message: Follow) {
```
Comment (bonustrack) [inline src/sign/index.ts:112]: Why not doing like this?
```suggestion
  async follow(web3: Web3Provider | Wallet, address: string, message: Follow) {
```

Comment (samuveth) [reply src/sign/index.ts:112]: I don't see when we would need `Web3Provider` there?

Comment (bonustrack) [reply src/sign/index.ts:112]: Someone may want to sign using his wallet, not a meta action

Comment (bonustrack) [reply src/sign/index.ts:112]: But yea is an edge case, doesnt matter much actually

**Review by bonustrack — APPROVED** [review]
Looks good

**Pull request merged**